test/torch/tensors/test_additive_shared.py

```diff
@@ -356,6 +356,30 @@ def test_fixed_precision_and_sharing(workers):
     assert (y == (t + t)).all()
 
 
+def test_remote_fixed_precision_and_sharing(workers):
+    bob, alice, james = (workers["bob"], workers["alice"], workers["james"])
+
+    t = torch.tensor([1, 2, 3, 4.0])
+    ptr = t.send(james)
+
+    x = ptr.fix_prec().share(bob, alice)
+
+    y = x + x
+
+    y = y.get().get().float_prec()
+    assert (y == (t + t)).all()
+
+    t = torch.tensor([1, 2, 3, 4.0])
```

Comment:
Can you put the two tests here as separate tests? Having them separate will help with debugging down the line (right now if the first one fails the second one doesn't run)

Reply:
Done!